src/main/java/frc/robot/subsystems/examplearm/ExampleArmCommandsBuilder.java

```diff
@@ -0,0 +1,40 @@
+package frc.robot.subsystems.examplearm;
+
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.InstantCommand;
+import edu.wpi.first.wpilibj2.command.RunCommand;
+import frc.utils.utilcommands.LoggedDashboardCommand;
+
+public class ExampleArmCommandsBuilder {
+
+	private final ExampleArm arm;
+
+	public ExampleArmCommandsBuilder(ExampleArm arm) {
+		this.arm = arm;
+	}
+
+	public Command moveToPosition(Rotation2d position) {
+		return arm.asSubsystemCommand(new InstantCommand(() -> arm.setTargetPosition(position)), "Move to position: " + position);
```

Comment:
```suggestion
		return arm.asSubsystemCommand(new InstantCommand(() -> arm.setTargetPosition(position)), "Move example arm to position: " + position);
```
and in all the other ones